lua/lualine/components/aerial.lua

```diff
@@ -55,12 +55,20 @@ local default_options = {
 
 local function color_icon(symbol_kind, icon, colored)
   if colored then
-    return string.format("%%#%s#%s%%##", "Aerial" .. symbol_kind .. "Icon", icon)
+    return string.format("%%#%s#%s%%##", "Aerial" .. symbol_kind .. "Icon", icon.." ")
   else
     return icon
```

Comment:
By adding the space here, but not on line 60, there won't be a space between the icon and the symbol name when color is disabled. I'd prefer to keep the spacing logic where it is, unless there's a reason to change it?

Reply:
That is a good point.  I wanted to handle the case where the syntax highlighting has a background color.  I think it might be better to change the status line string from `"%%#%s#%s%%##"` to `"%%#%s#%s"`.

![image](https://user-images.githubusercontent.com/46944641/193959455-9047d3ba-24ab-4841-a647-64cce99e9bdd.png).

I will revert the spacing change and instead update the status line string.

After revisiting this, I think it would also be helpful to create a new set of highlights beginning with 
`"LuaLineAerial" .. symbol_kind`  

that links to the original, so that you can independently control how the syntax appears in lualine and in the aerials window.  I'll update this PR with that change to illustrate.